kvm/src/lib.rs

```diff
@@ -1381,8 +1381,10 @@ mod tests {
             0x04, b'0', /* add $'0', %al */
             0xee, /* out %al, %dx */
             0xec, /* in %dx, %al */
-            0xc6, 0x06, 0x00, 0x20, 0x00, /* movl $0, (0x2000) */
-            0x8a, 0x16, 0x00, 0x20, /* movl (0x2000), %dl */
+            0xc6, 0x06, 0x00, 0x80, 0x00, /* movl $0, (0x8000); This generates a MMIO Write.*/
+            0x8a, 0x16, 0x00, 0x80, /* movl (0x8000), %dl; This generates a MMIO Read.*/
+            0xc6, 0x06, 0x00, 0x20,
+            0x00, /* movl $0, (0x2000); Dirty one page in guest mem. */
```

Comment:
Please shorten the comment so that it fits on the same line with the corresponding asm, it's a bit confusing.

Reply:
I didn't find a way to shorten it and keep the message. The problem is that without the comments it is harder to know which instruction is trigger which KVM exit and why do we want it. It took me a while to discover which was which. Do you have any suggestions? :(

Reply:
Found a way. I will disable rustfmt for that code snippet. In this case I believe cargo fmt does not improve readability.